### PR TITLE
Respect user specified new-line character in split-string

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/41322

### CODE REVIEW

File: src/EditorFeatures/CSharp/SplitStringLiteral/SplitStringLiteralCommandHandler.cs [hunk 1]
```diff
@@ -99,7 +99,8 @@ public bool ExecuteCommandWorker(ReturnKeyCommandArgs args)
             }
 
             var parsedDocument = ParsedDocument.CreateSynchronously(document, CancellationToken.None);
-            var options = subjectBuffer.GetIndentationOptions(_editorOptionsService, parsedDocument.LanguageServices, explicitFormat: false);
+            var formattingOptions = subjectBuffer.GetLineFormattingOptions(_editorOptionsService, explicitFormat: false);
```
Comment (tmat) [inline src/EditorFeatures/CSharp/SplitStringLiteral/SplitStringLiteralCommandHandler.cs:102]: >formattingOptions [](http://example.com/codeflow?start=16&length=17)

The formatting options are already included in IndentationOptions

Comment (CyrusNajmabadi) [reply src/EditorFeatures/CSharp/SplitStringLiteral/SplitStringLiteralCommandHandler.cs:102]: it is not afaict :)

Comment (tmat) [reply src/EditorFeatures/CSharp/SplitStringLiteral/SplitStringLiteralCommandHandler.cs:102]: `IndentationOptions.FormattingOptions.LineFormatting`?

Comment (CyrusNajmabadi) [reply src/EditorFeatures/CSharp/SplitStringLiteral/SplitStringLiteralCommandHandler.cs:102]: d'oh.  i'm dumb.  fixing.

**Comment by CyrusNajmabadi** [comment]
@tmat  ptal.

**Review by tmat — APPROVED** [review]
:shipit:

**Pull request merged**